client/client.go

```diff
@@ -504,6 +508,12 @@ type (
 		// NOTE: Experimental
 		UpdateWorkflowWithOptions(ctx context.Context, request *UpdateWorkflowWithOptionsRequest) (WorkflowUpdateHandle, error)
 
+		// GetWorkflowUpdateHandle creates a handle to the referenced update
+		// which can be polled for an outcome. Note that runID is optional and
+		// if not specified the most recent runID will be used.
+		// NOTE: Experimental
+		GetWorkflowUpdateHandle(ref WorkflowUpdateRef) WorkflowUpdateHandle
```

Comment:
```suggestion
		GetWorkflowUpdateHandle(options GetWorkflowUpdateOptions) WorkflowUpdateHandle
```

I know it's a bit wordier, but reads better with regards to other calls here

Reply:
If we're going that direction, wouldn't it be `GetWorkflowUpdateHandleOptions`? I'm not sure what you're suggesting we be consistent with since this is just a constructor. Arguably it should just be a free function.

Reply:
I thought the suggestion was to rename `WorkflowUpdateRef` -> `GetWorkflowUpdateOptions`.

Reply:
That is the suggestion - But if the pattern is Get${THING} takes an arg of type ${THING}Options (and I _think_ that's pattern being referred to as "other calls here") then $THING here is WorkflowUpdateHandle, not WorkflowUpdate

Reply:
ah I see your point. Personally I am fine with `GetWorkflowUpdateHandleOptions`